lib/graphql/schema/type_reducer.rb

```diff
@@ -1,10 +1,19 @@
 # frozen_string_literal: true
+
+require 'active_support/inflector'
+
 module GraphQL
   class Schema
-    module ReduceTypes
+    class TypeReducer
```

Comment:
Hmm I don't [see any third-party code](https://github.com/search?utf8=%E2%9C%93&q=ReduceTypes+language%3ARuby+extension%3Arb&type=Code&ref=advsearch&l=Ruby&l=) depending on this, but still, I've been surprised at my capacity to break code by changing things which I failed to mark `@api private` early on.

It seems like this _could_ be implemented in a backwards-compatible way, right? Adding `camelize: false` to `.reduce` and adding `camelize:` to all the functions it calls? It's definitely more verbose, but are there other downsides to that approach?